apps/src/craft/craft.js

```diff
@@ -281,6 +281,10 @@ Craft.init = function (config) {
       if (!config.level.showPopupOnLoad) {
         Craft.initializeAppLevel(config.level);
       }
+
+      // Adjust visualizationColumn width.
+      var visualizationColumn = document.getElementById('visualizationColumn');
+      visualizationColumn.style.width = this.nativeVizWidth + 'px';
```

Comment:
Looking at this now.

On Fri, Dec 4, 2015 at 12:47 AM, Andrew Oberhardt <notifications@github.com>
wrote:

> In apps/src/craft/craft.js
> <https://github.com/code-dot-org/code-dot-org/pull/5850#discussion_r46660033>
> :
>
> > @@ -281,6 +281,10 @@ Craft.init = function (config) {
> >        if (!config.level.showPopupOnLoad) {
> >          Craft.initializeAppLevel(config.level);
> >        }
> > +
> > +      // Adjust visualizationColumn width.
> > +      var visualizationColumn = document.getElementById('visualizationColumn');
> > +      visualizationColumn.style.width = this.nativeVizWidth + 'px';
>
> Good catch. You're right, this messed with resizing in the non-sharing
> view. I updated this to only apply to sharing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/5850/files#r46660033>.
>